flocker/testtools/__init__.py

```diff
@@ -684,35 +682,6 @@ def remove_image():
         return d.addCallback(lambda ignored: tag)
 
 
-def skip_on_broken_permissions(test_method):
```

Comment:
I'm -0 about this change. If we set up something with packer for doing development on OS X (and we should do that), then we'll probably still need this or something like it. 

Or, another way, removing this seems like a separate thing to removing the tutorial box set-up code.